yt_dlp/extractor/digitalconcerthall.py

```diff
@@ -0,0 +1,201 @@
+            'client_secret': self._CLIENT_SECRET,
+        }
+        try:
+            token_response = self._download_json(
+                self._OAUTH_URL,
+                None, 'Obtaining token', data=urlencode_postdata(data), headers={
+                    'Content-Type': 'application/x-www-form-urlencoded',
+                })
+        except ExtractorError as e:
+            msg = 'Unable to obtain token'
+            if isinstance(e.cause, compat_HTTPError) and e.cause.code == 401:
+                resp = self._parse_json(e.cause.read().decode(), None, fatal=False)
+                if resp:
+                    error = resp.get('extra_info') or resp.get('error_description') or resp.get('error')
+                    if error:
+                        msg += ': ' + error
+            raise ExtractorError('Unable to obtain token: ' + msg)
+        self.debug_out("token_response: " + json.dumps(token_response))
+        self._ACCCESS_TOKEN = token_response.get('access_token')
+        # now login
+        data = {
+            'grant_type': 'password',
+            'username': username,
+            'password': password,
+        }
+        try:
+            self._download_json(
+                self._OAUTH_URL,
+                None, 'Logging in', data=urlencode_postdata(data), headers={
+                    'Content-Type': 'application/x-www-form-urlencoded',
+                    'Referer': self._LOGIN_URL,
+                    'Authorization': 'Bearer ' + self._ACCCESS_TOKEN
+                })
+        except ExtractorError as e:
+            msg = 'Unable to login'
+            if isinstance(e.cause, compat_HTTPError) and e.cause.code == 401:
+                resp = self._parse_json(e.cause.read().decode(), None, fatal=False)
+                if resp:
+                    error = resp.get('extra_info') or resp.get('error_description') or resp.get('error')
+                    if error:
+                        msg += ': ' + error
+            raise ExtractorError('Unable to log in: ' + msg)
+
+    def _real_initialize(self):
+        self._login()
+
+    def _real_extract(self, url):
+        MAX_TITLE_LENGTH = 128
+        language, video_id = re.match(self._VALID_URL, url).groups()
+        if not language:
+            language = 'en'
+        self.debug_out("url: " + url + " video_id: " + video_id + " language: " + language)
+        webpage = self._download_webpage(url, video_id)
+        playlist_title = self._html_search_regex(r'<title>(.+?)</title>', webpage, 'title') or \
+            self._og_search_title(webpage)
+        self.debug_out("playlist_title: " + playlist_title)
+
+        # use the API to get other information about the concert
+        vid_info_dict = self._download_json(
+            'https://api.digitalconcerthall.com/v2/concert/'
+            + video_id, video_id, headers={'Accept': 'application/json',
+                                           'Accept-Language': language})
```

Comment:
```suggestion
        vid_info_dict = self._download_json(
            f'https://api.digitalconcerthall.com/v2/concert/{video_id}', video_id,
            headers={
                'Accept': 'application/json',
                'Accept-Language': language
            })
```
Also, change the var name to something else. We use `info_dict` generally to refer to the return value of extractors. So this is a bit confusing